### ISSUE TITLE
[Outdated] natron + openfx

### ISSUE THREAD

**Issue by a61kt** [chaotic-aur/packages#1461]
### If available, link to the latest build

https://github.com/NatronGitHub/Natron/releases/tag/v2.4.3
https://github.com/NatronGitHub/openfx-arena/releases/tag/Natron-2.4.3
https://github.com/NatronGitHub/openfx-gmic/releases/tag/Natron-2.4.3
https://github.com/NatronGitHub/openfx-io/releases/tag/Natron-2.4.3
https://github.com/NatronGitHub/openfx-misc/releases/tag/Natron-2.4.3

### Package name

natron,openfx-io,openfx-gmic,openfx-misc,openfx-arena

### Latest build

2.4.3

### Latest version available

2.4.2

### Have you tested if the package builds in a clean chroot?

- [ ] Yes

**Comment by dr460nf1r3** [comment]
I checked the corresponding AUR pages, the packages are currently flagged out of date there. Since we build from the AUR, this is nothing we would fix ourselves unless it is a package we maintain. Once the actual maintainer updates the version, the packages would rebuild themselves automatically unless an issue prevents the build (this would show up in the build log, which should be checked prior to creating an issue). 

I did check the build logs though and found `natron` to be missing the dependency `openfx-io`, which is also flagged out of date and fails to build. Already reported this to the maintainer on 27.2.22, no answer until now.
